activesupport/lib/active_support/testing/time_helpers.rb

```diff
@@ -134,9 +134,9 @@ def travel_to(date_or_time)
           now = date_or_time.to_time.change(usec: 0)
         end
 
-        simple_stubs.stub_object(Time, :now, now)
-        simple_stubs.stub_object(Date, :today, now.to_date)
-        simple_stubs.stub_object(DateTime, :now, now.to_datetime)
+        simple_stubs.stub_object(Time, :now) { at(now.to_i) }
+        simple_stubs.stub_object(Date, :today) { parse(now.to_date.to_s) }
+        simple_stubs.stub_object(DateTime, :now) { parse(now.to_datetime.to_s) }
```

Comment:
> seems that Time zone information is lost this way

Ah, that would make sense since a [Julian Day][1] is in [Universal Time][2].

[1]: https://en.wikipedia.org/wiki/Julian_day
[2]: https://en.wikipedia.org/wiki/Universal_Time